analytics-core/src/test/java/com/liferay/analytics/client/android/api/impl/AnalyticsClientImplTest.kt

```diff
@@ -0,0 +1,126 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.analytics.client.android.api.impl
+
+import com.google.gson.reflect.TypeToken
+import com.liferay.analytics.client.android.model.AnalyticsEventsMessageModel
+import com.liferay.analytics.client.android.util.JSONParser
+import com.liferay.analytics.model.AnalyticsEventsMessage
+import okhttp3.MediaType
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import okhttp3.RequestBody
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Test
+import org.mockito.Mockito
+import java.text.SimpleDateFormat
+import java.util.*
+import java.util.concurrent.TimeUnit
+
+/**
+ * @author Igor Matos
+ * @author Allan Melo
+ */
+class AnalyticsClientImplTest {
+
+	private val _analyticsClientImpl = Mockito.spy(AnalyticsClientImpl::class.java)
+
+	private lateinit var _userId: String
+
+	@Before
+	fun setUp() {
+		Mockito.`when`(_analyticsClientImpl.analyticsGatewayHost)
+			.thenReturn("192.168.108.90")
```

Comment:
mmm, can you a global server?

Reply:
We don't have a global server for this test yet. We can use the global server to send events, but this server doesn't have any endpoint to test if the event was sent correctly, so temporarily we enabled locally the query on Cassandra to do these tests.

Reply:
Ok!